website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -2666,6 +2666,11 @@ infer the correct schema based on the members of the class.
 In Python you can use the following set of classes to represent the purchase schema. Beam will automatically infer the correct schema based on the members of the class.
 {{< /paragraph >}}
 
+{{< paragraph class="language-go" >}}
+In Go, schema encoding used by default for struct types, with Exported fields becoming part of the schema.
```

Comment:
Is there a word missing here?

Reply:
Nice catch! the missing word was "is".  ...encoding is used by....